src/lib.rs

```diff
@@ -149,15 +149,19 @@ impl ChangesStream {
     ///    changes.run();
     /// # }
     /// ```
-    pub fn run(mut self) {
+    pub fn run(mut self, mut lp: Option<tokio_core::reactor::Core>) {
```

Comment:
So, while technically this works, It's a bit annoying to use. The common pattern is to use two functions instead:

    pub fn run(mut self)
    pub fn run_with(mut self, mut lp: tokio_core::reactor::Core)